splice_machine/src/main/java/com/splicemachine/derby/stream/control/ControlDataSetProcessor.java

```diff
@@ -404,4 +422,45 @@ public TableChecker getTableChecker(String schemaName, String tableName, DataSet
     @Override public void incrementOpDepth() { }
     @Override public void decrementOpDepth() { }
     @Override public void resetOpDepth() { }
+
+    @Override
+    public <V> DataSet<ExecRow> readKafkaTopic(String topicName, OperationContext context) throws StandardException {
+//        Properties props = new Properties();
+//        props.put("bootstrap.servers", SIDriver.driver().getConfiguration().getKafkaBootstrapServers());
+//        props.put("enable.auto.commit", false);
+//        props.put("auto.commit.interval.ms", "1000");
+//        props.put("session.timeout.ms", "30000");
+//        props.put("auto.offset.reset", "latest");
+//
+//        props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, IntegerDeserializer.class.getName());
+////        props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, ExternalizableDeserializer.class.getName());
+//        //ExternalizableDeserializer was implemented in package com.splicemachine.derby.stream.spark
+//
+//        KafkaConsumer<Integer, Externalizable> consumer = new KafkaConsumer<Integer, Externalizable>(props);
+//        consumer.subscribe(Arrays.asList(topicName));
+//
+//        try {
+//            ConsumerRecords<Integer, Externalizable> msgList = consumer.poll(1000);
+//            for (ConsumerRecord<Integer, Externalizable> record : msgList) {
+//                if (LOG.isInfoEnabled())
+//                    LOG.info("KAFKA==receive: key = " + record.key() + ", value = " + record.value() + " offset===" + record.offset());
+//            }
+//
+//            PairDataSet<Integer, InputStream> streamSet;
+//
+//            consumer.commitAsync();
+//
+//            Dataset<Row> table = null;
+//        } catch (CommitFailedException e){
+//            throw new StandardException();
+//        }
+//        finally {
+//            try {
+//                consumer.commitSync();
+//            }finally {
+//                consumer.close();
+//            }
+//        }
```

Comment:
Remove them?

Reply:
Fixed in new commit [e15bf8e](https://github.com/splicemachine/spliceengine/pull/3391/commits/e15bf8e4ed58fce49d5a3048e6bca875bad887bd)